config/default/kustomization.yaml

```diff
@@ -1,71 +1,13 @@
+---
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+
 # Adds namespace to all resources.
 namespace: openshift-logging
 
-# Value of this field is prepended to the
-# names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
-# Note that it should also match with the prefix (text before '-') of the namespace
-# field above.
-# namePrefix: clo-
-
-# Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
-
 bases:
 - ../crd
-- ../rbac
 - ../manager
-- ../scheduling
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in 
```

Comment:
Consider leaving these in for hints when we want to add webhooks

Reply:
ok